crates/sui-bridge/src/types.rs

```diff
@@ -467,6 +467,30 @@ pub struct EthLog {
     pub log: Log,
 }
 
+/// Check if the bridge route is valid
+/// Only mainnet can bridge to mainnet, other than that we do not care.
+pub fn is_route_valid(one: BridgeChainId, other: BridgeChainId) -> bool {
+    if one.is_sui_chain() && other.is_sui_chain() {
+        return false;
+    }
+    if !one.is_sui_chain() && !other.is_sui_chain() {
+        return false;
+    }
```

Comment:
are those needed at all?

Reply:
yes, we don't want sui to bridge to sui. No?